FEM/fem_ele_vec.cpp

```diff
@@ -1634,8 +1599,10 @@ void CFiniteElementVec::GlobalAssembly_RHS()
 				else if (pcs->ExcavMaterialGroup > -1)
 				{
 					double const* ele_center(elem->GetGravityCenter());
-					if ((GetCurveValue(pcs->ExcavCurve, 0, aktuelle_zeit, &valid) + pcs->ExcavBeginCoordinate)
-					    < (ele_center[pcs->ExcavDirection]))
+					const double expected_coordinate =
```

Comment:
same here

Reply:
Just took a loot. Here the if-condition is different from that in the new function.